### ISSUE TITLE
404 404: /base/dist/traceur

### ISSUE THREAD

**Issue by avi1000** [angular/angular-cli#1336]
1. OS - Windows 7 64bit

2.node: v4.4.7

3. angular-cli: 1.0.0-beta.9

Steps:
I'm trying to run ng test --build=false but getting the following error:
13 07 2016 16:24:06.651:WARN [karma]: No captured browser, open http://localhost:9876/
13 07 2016 16:24:06.651:INFO [karma]: Karma v0.13.22 server started at http://localhost:9876/
13 07 2016 16:24:06.669:INFO [launcher]: Starting browser Chrome
13 07 2016 16:24:07.762:INFO [Chrome 51.0.2704 (Windows 8.1 0.0.0)]: Connected on socket /#BZh6DNcg9qNNVzNYAAAA with id 90618129
13 07 2016 16:24:09.570:WARN [web-server]: 404: /base/dist/traceur
Missing error handler on `socket`.

**Comment by avi1000** [comment]
It looks like if there are no code in ALL the specs, I got this error.

**Comment by neilkyoung** [comment]
I also get this when running my ng test command except my spec files are not empty. I am wondering if this is also to do with my TypeError error:

```
20 07 2016 16:36:19.749:WARN [karma]: No captured browser, open http://localhost:9876/
20 07 2016 16:36:19.772:INFO [karma]: Karma v0.13.22 server started at http://localhost:9876/
20 07 2016 16:36:19.780:INFO [launcher]: Starting browser Chrome
20 07 2016 16:36:21.684:INFO [Chrome 51.0.2704 (Mac OS X 10.11.1)]: Connected on socket /#75sMDWNDg_Q0CmVUAAAA with id 88012910
20 07 2016 16:36:24.439:WARN [web-server]: 404: /base/dist/traceur
Missing error handler on `socket`.
TypeError: (msg || "").replace is not a function
    at /Users/substance/Sites/angular-2-testing/node_modules/karma/lib/reporter.js:45:23
    at onBrowserError (/Users/substance/Sites/angular-2-testing/node_modules/karma/lib/reporters/base.js:58:60)
    at null.<anonymous> (/Users/substance/Sites/angular-2-testing/node_modules/karma/lib/events.js:13:22)
    at emitTwo (events.js:87:13)
    at emit (events.js:172:7)
    at onKarmaError (/Users/substance/Sites/angular-2-testing/node_modules/karma/lib/browser.js:95:13)
    at Socket.<anonymous> (/Users/substance/Sites/angular-2-testing/node_modules/karma/lib/events.js:13:22)
    at emitOne (events.js:82:20)
    at Socket.emit (events.js:169:7)
    at Socket.onevent (/Users/substance/Sites/angular-2-testing/node_modules/socket.io/lib/socket.js:335:8)
    at Socket.onpacket (/Users/substance/Sites/angular-2-testing/node_modules/socket.io/lib/socket.js:295:12)
    at Client.ondecoded (/Users/substance/Sites/angular-2-testing/node_modules/socket.io/lib/client.js:193:14)
    at Decoder.Emitter.emit (/Users/substance/Sites/angular-2-testing/node_modules/component-emitter/index.js:134:20)
    at Decoder.add (/Users/substance/Sites/angular-2-testing/node_modules/socket.io-parser/index.js:247:12)
    at Client.ondata (/Users/substance/Sites/angular-2-testing/node_modules/socket.io/lib/client.js:175:18)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at Socket.onPacket (/Users/substance/Sites/angular-2-testing/node_modules/engine.io/lib/socket.js:101:14)
    at emitOne (events.js:77:13)
    at WebSocket.emit (events.js:169:7)
    at WebSocket.Transport.onPacket (/Users/substance/Sites/angular-2-testing/node_modules/engine.io/lib/transport.js:104:8)
    at WebSocket.Transport.onData (/Users/substance/Sites/angular-2-testing/node_modules/engine.io/lib/transport.js:115:8)
20 07 2016 16:36:26.455:WARN [Chrome 51.0.2704 (Mac OS X 10.11.1)]: Disconnected (1 times)
```

**Comment by filipesilva** [comment]
Closed as issue was made obsolete by #1455.

**Comment by narainsagar** [comment]
Hi, I am using `"angular-cli": "^1.0.0-beta.10"` version with "@angular" "2.0.0-rc.5" and I am getting the same error... not getting any fix... :(
